pkg/cli/admin/ocpcertificates/trustpurge/trustpurge.go

```diff
@@ -40,18 +41,69 @@ type RemoveOldTrustRuntime struct {
 
 	Printer printers.ResourcePrinter
 
+	cachedSecretCerts map[string][]*cachedSecretCert
+
 	genericclioptions.IOStreams
 }
 
+type cachedSecretCert struct {
+	namespaceName string
+	cert          *x509.Certificate
+}
+
+func newCachedSecretCert(namespace, name string, certPEM []byte) (*cachedSecretCert, error) {
+	if len(certPEM) == 0 {
+		return nil, nil
+	}
+
+	secretNamespaceName := namespace + "/" + name
```

Comment:
in other commands we've used `resource-type/name[namespace]`.  not universal, but at least it fails cleanly on a copy/paste.